.goreleaser.yaml

```diff
@@ -0,0 +1,62 @@
+# TODO @JPERRY REMOVE THESE LINES AND ENSURE THE 'GITHUB_TOKEN' token is populated in the repo.
+# .goreleaser.yaml
+env_files:
+  github_token: ~/.path/to/my/github_token
+
+
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+
+universal_binaries:
+  - replace: false
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    dir: src
+    ldflags:
+      - -s -w -X github.com/defenseunicorns/zarf/src/config.CLIVersion={{.Version}}
+    goarch:
+      - amd64
+      - arm64
+archives:
+  -
+    format: binary
+    replacements:
+      darwin: Darwin
+      linux: Linux
+
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  use: github-native
+
+release:
+  github:
+    owner: defenseunicorns
+    name: zarf
+  prerelease: auto
+  mode: append
+
+brews:
+  - name: zarf
+    tap:
+      owner: defenseunicorns
+      name: homebrew-tap
+      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
+    commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
+    homepage: "https://zarf.dev/"
+    description: "An airgap delivery tool!"
+
+blobs:
```

Comment:
This will drop all the same binaries in S3 as the prior workflow method, correct?

Reply:
That is the end goal! 

This comment made me realize we accidentally stopped building the init-packages in the current PR since GoReleaser doesn't handle that for us. That will be fixed with the next commit, thanks!

Reply:
That is the end goal! 

This comment made me realize we accidentally stopped building the init-packages in the current PR since GoReleaser doesn't handle that for us. That will be fixed with the next commit, thanks!